source/authentication/email-password.txt

```diff
@@ -212,30 +217,38 @@ The following table describes the potential values of this field:
 
    * - ``pending``
 
-     - App Services changes the user's confirmation status to ``pending``.
+     - App Services changes the user's confirmation status to ``Pending Confirmation``.
```

Comment:
I wonder if we want to add something about the `Pending` case relative to the email being registered, as in attempting to re-register the user while the confirmation is `Pending` will fail because the user's email is in the system, even though their account hasn't been confirmed. I've seen people get tripped up on that before.

Maybe we want to point them to the [Admin API endpoint where they can list pending users](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/docsp-27595/admin/api/v3/#tag/users/operation/adminListPendingUsers), to check for a pending user account in that case?

Technically, I suppose that applies to both "Send a user confirmation email" and "confirm with a custom function" so maybe there should be a separate section at the top level of the User Confirmation section describing the three possible states, implications for the email being "in use", and how a dev can use the Admin API to check for a pending user?